sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/CustomValidationPolicy.java

```diff
@@ -42,15 +43,15 @@ public Mono<HttpResponse> process(
                 .map(CustomValidationPolicy::ungzip);
         asyncBytes.subscribe(
             value -> {
-                ObjectMapper objectMapper = createObjectMapper();
-                try (MappingIterator<TelemetryItem> i =
-                         objectMapper.readerFor(TelemetryItem.class).readValues(value)) {
-                    while (i.hasNext()) {
-                        actualTelemetryItems.add(i.next());
+                List<byte[]> splitBytesByNewline = splitBytesByNewline(value.getBytes());
```

Comment:
@alzimmermsft this is what i referred to earlier splitting original big byte[] into a list of byte[] per newline delimiter. 
if `readArray` works, i don't need to do this workaround.

Reply:
updated code to support newlines deserialization https://github.com/Azure/azure-sdk-for-java/commit/c9502466e69f264d925c7692d055c4291afc5638